config/routes.rb

```diff
@@ -3,10 +3,11 @@
   # See how all your routes lay out with "rake routes".
 
   # You can have the root of your site routed with "root"
-  # root 'welcome#index'
+  root 'items#index'
 
+  get "/new_item" => "items#new"
```

Comment:
Please review Rails routing: http://guides.rubyonrails.org/routing.html#crud-verbs-and-actions

These routes don't follow typical REST convention.